### PR TITLE
Document changelog variable in Manual.md

### PR DESCRIPTION
**Pull request by meator**
[ci skip]

### CODE REVIEW

File: Manual.md [hunk 1]
```diff
@@ -743,6 +743,8 @@ used.
 
 - `fetch_cmd` Executable to be used to fetch URLs in `distfiles` during the `do_fetch` phase.
 
+- `changelog` A string pointing to the `upstream` changelog.
```
Comment (ericonr) [inline Manual.md:746]: Not sure "upstream" should be formatted as code. I think this should also document that raw text files are preferred.

**Review by ericonr — COMMENTED** [review]
Commit should be closer to `Manual.md: document changelog`

Thanks!

File: Manual.md (hunk 1 above)
Comment (paper42) [reply Manual.md:746]: Let's also be clear that the string is a URL (not a filename to changelog in the distfiles).

Comment (meator) [reply Manual.md:746]: I just copied description for homepage and changed it to changelog. I'll also change homepage to make it consistent.

**Pull request merged**